<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>1646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <!-- Menu Widget: Contains Exit Button -->
    <item>
     <widget class="QWidget" name="menuWidget">
      <layout class="QGridLayout" name="gridLayout_TileArea">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
    <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonExit">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(164, 0, 0, 0.5);
border-radius: 4px;
border: 2px solid rgba(255,255,255,0.5);</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/power-hot.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Power Widget: Contains Shutdown, Reboot and Cancel Controls -->
    <item>
     <widget class="QWidget" name="horizontalWidgetPower">
      <layout class="QHBoxLayout" name="horizontalLayoutPower">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonShutdown">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(239, 41, 41, 0.5);
border-radius: 4px;
border: 2px solid rgba(255,255,255,0.5);</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/power-hot.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonReboot">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(252, 175, 62, 0.5);
border-radius: 4px;
border: 2px solid rgba(255,255,255,0.5);</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/reboot-hot.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(32, 74, 135, 0.5);
border-radius: 4px;
border: 2px solid rgba(255,255,255,0.5);</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/back-hot.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
